packages/aws-cdk-lib/core/lib/duration.ts

```diff
@@ -62,6 +62,9 @@ export class Duration {
   /**
    * Parse a period formatted according to the ISO 8601 standard
    *
+   * Days are the largest ISO duration supported, i.e.,
+   * months and years are not supported.
```

Comment:
Weeks are also not supported:

```suggestion
   * weeks, months and years are not supported.
```

Reply:
done 👍🏼